tests/test_execute.py

```diff
@@ -130,3 +130,30 @@ def task_run_jl_script():
     session = main({"paths": tmp_path})
 
     assert session.exit_code == 0
+
+
+@needs_julia
+@pytest.mark.end_to_end
+def test_check_passing_cmd_line_options(tmp_path):
+    task_source = """
+    import pytask
+
+    @pytask.mark.julia("-t 4")
+    @pytask.mark.depends_on("script.jl")
+    @pytask.mark.produces("out.txt")
+    def task_run_jl_script():
+        pass
+
+    """
+    tmp_path.joinpath("task_dummy.py").write_text(textwrap.dedent(task_source))
+
+    julia_script = """
+    write("out.txt", "So, so you think you can tell heaven from hell?")
+    @assert Threads.nthreads() == 1
+    """
+    tmp_path.joinpath("script.jl").write_text(textwrap.dedent(julia_script))
+
+    os.chdir(tmp_path)
+    session = main({"paths": tmp_path})
```

Comment:
You dont need to switch paths and better use the runner patter from here

https://github.com/pytask-dev/pytask-julia/blob/2b15b0745214ae45e5b190bdb63e0decaeb6bcc8/tests/test_parallel.py#L58